packages/streams/nmea0183-signalk.js

```diff
@@ -31,6 +31,7 @@
 
 const Transform = require('stream').Transform
 const Parser = require('@signalk/nmea0183-signalk')
+const utils = require('@signalk/nmea0183-utilities')
```

Comment:
Doesn't this need a dependency from @signalk/streams to @signalk/nmea0183-utilities? @signalk/nmea0183-signalk has it, so it is already a transient dependency, but this makes it direct.

Reply:
Furthermore it must be ^0.8.0 as you said, nmea0183-signalk doesn't need that.